xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -332,7 +332,15 @@
         <summary>Gets a value that indicates whether the collection is read only.</summary>
         <value>
           <see langword="true" /> in all cases.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+A collection that is read-only does not allow the addition or removal of elements after the collection is created. Note that read-only in this context does not indicate whether individual elements of the collection can be modified, since the <xref:System.Collections.Generic.ICollection%601> interface only supports addition and removal operations. For example, the <xref:System.Collections.Generic.ICollection%601.IsReadOnly%2A> property of an array that is cast or converted to an <xref:System.Collections.Generic.ICollection%601> object returns `true`, even though individual array elements can be modified.
```

Comment:
This is an inherited remark. Does it also make sense in this API?

Reply:
Yup, this is relevant in this API